extensions/kubeapi/projects/projects.go

```diff
@@ -0,0 +1,30 @@
+package projects
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	Admin                            = "admin"
+	StandardUser                     = "user"
+	DefaultNamespace                 = "fleet-default"
+	RancherNamespace                 = "cattle-system"
+	LocalCluster                     = "local"
+	Projects                         = "projects"
+	ProjectIDAnnotation              = "field.cattle.io/projectId"
+	MemberBindingLabel               = "membership-binding-owner"
+	ProjectScopedLabel               = "cattle.io/project-scoped"
+	ProjectScopedLabelValue          = "original"
+	ProjectScopedNamespaceLabelValue = "copy"
+	NormanCreatorLabel               = "cattle.io/creator"
+	NormanCreatorLabelValue          = "norman"
```

Comment:
As these are not frequently used as the others, should we limit these to the test helpers?